File: backend2/DataAccess/DaoGenerated/システム共有/tt_afatenalogDto.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: DTO定義
//             宛名番号検索履歴テーブル
// 作成日　　: 2024.07.23
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afatenalogDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afatenalog";
		public string userid { get; set; }                        //ユーザーID
		public string kinoid { get; set; }                        //機能ID
		public string atenano { get; set; }                       //宛名番号
		public DateTime syoridttm { get; set; }                   //処理日時
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afatenalogDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             宛名番号検索履歴テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afatenalogDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afatenalog"
        Public Property userid As String                        'ユーザーID
        Public Property kinoid As String                        '機能ID
        Public Property atenano As String                       '宛名番号
        Public Property syoridttm As Date                   '処理日時
    End Class
End Namespace
